crates/turborepo-lib/src/run/scope/filter.rs

```diff
@@ -148,6 +148,7 @@ impl<'a, T: PackageChangeDetector> FilterResolver<'a, T> {
             // return all packages in the workspace
             self.pkg_graph
                 .workspaces()
+                .filter(|(name, _)| **name != WorkspaceName::Root)
```

Comment:
Nit: generally, I think it's better to use `matches!` instead of the `Eq` implementation here. I assume that perf would be pretty identical, but since `matches!` expands to a the match statement we want directly I think it's equivalent to `#[inline]` the eq function.
```suggestion
                .filter(|(name, _)| matches!(name, WorkspaceName::Other(_)))
```

Reply:
I'm fine with `matches!` here, the `**name` construction is definitely awkward. The other thought I had was implementing an `.is_root()` method on the enum, but decided that was maybe slightly overkill.